test/harvest/test_dimensions.py

```diff
@@ -56,9 +56,8 @@ def f(*args, **kwargs):
     monkeypatch.setattr(dimensions, "orcid_publications", f)
 
 
```

Comment:
There are a bunch of changes related to using the `snapshot` fixture, which cuts down on boilerplate a little bit.